<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_main</class>
 <widget class="QWidget" name="win_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>win_main</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ax">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ay">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_az">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gx">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gz">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gy">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
